src/index.js

```diff
@@ -0,0 +1,945 @@
+    },
+  },
+  {
+    class: ['AltLeft', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['Space', 'special'],
+    eng: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+    ru: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+  },
+  {
+    class: ['AltRight', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['ArrowLeft', 'special'],
+    eng: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+    ru: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+  },
+  {
+    class: ['ArrowDown', 'special'],
+    eng: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+    ru: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+  },
+  {
+    class: ['ArrowRight', 'special'],
+    eng: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+    ru: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+  },
+  {
+    class: ['ControlRight', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+];
+
+/* global document, localStorage */
+
+const rows = [keyRow1, keyRow2, keyRow3, keyRow4, keyRow5];
+
+const myBody = document.querySelector('body');
+
+// create wrapper for keyboard
+myBody.innerHTML =
+  '<section class="wrapper"><textarea id="text" placeholder="Write smth!" rows="15"></textarea><section class="keyboard"></section></section>';
+
+const createRow = keyRow => {
+  const row = document.createElement('div');
+  row.classList.add('row');
+
+  for (let i = 0; i < keyRow.length; i += 1) {
+    const myKey = document.createElement('div');
+    myKey.className = 'key';
+    myKey.classList.add(...keyRow[i].class);
+
+    myKey.innerHTML = `<span class="ru hide"><span class="shift up hide">${keyRow[i].ru.shiftUp}</span><span class="shift down hide">${keyRow[i].ru.shiftDown}</span></span><span class="eng"><span class="shift up">${keyRow[i].eng.shiftUp}</span><span class="shift down hide">${keyRow[i].eng.shiftDown}</span></span>`;
+    row.appendChild(myKey);
+  }
+  return row;
+};
+
+// creating keyboard
+const createKeyboard = rowsArr => {
+  for (let i = 0; i < rowsArr.length; i += 1) {
+    document.querySelector('.keyboard').appendChild(createRow(rowsArr[i]));
+  }
+};
+
+createKeyboard(rows);
+
+// ------------ KEYBOARD EVENTS ------------- //
+const keyboard = document.querySelector('.keyboard');
+const text = document.querySelector('#text');
+let shiftState = 'up';
+let lang = 'eng';
+
+function backspace(str, start, end) {
+  let result = '';
+  if (start === end) {
+    result = str.slice(0, start-1) + str.slice(start+1, str.length);
+  } else {
+    result = str.replace(str.slice(start, end), '');
+  }
+  return result;
+}
+
+function myDelete(str, start, end) {
```

Comment:
what is `myDelete` ? Please follow naming conversion for variables and functions. function name should describe what this function suppose to do, should started from verb, etc.

Reply:
fixed